util/utils.go

```diff
@@ -14,28 +14,21 @@
  * limitations under the License.
  */
 
-package prime
+package util
 
 import (
-	"net/url"
-	"os"
-
-	log "github.com/sirupsen/logrus"
+	"crypto/md5"
+	"fmt"
+	"strings"
 )
 
-var primeV1ApiBaseUrl = "https://api.prime.coinbase.com/v1"
+func GenerateUniqueId(params ...string) string {
+	return fmt.Sprintf("%x", md5.Sum([]byte(strings.Join(params, "-"))))
```

Comment:
Why not use the uuid package here?

Reply:
I moved the utils to the packages where they are being called and removed the calculate order, which I already moved to the Prime SDK